week_1/String/longest_common_prefix.js

```diff
@@ -0,0 +1,21 @@
+// Write a function to find the longest common prefix string amongst an array of strings.
+
+// If there is no common prefix, return an empty string "".
```

Comment:
Can we update this to use the jsdoc `example` tag?

https://jsdoc.app/tags-example.html